### ISSUE TITLE
Can we Implement the code on the new DWM3001C model?

### ISSUE THREAD

**Issue by zyliuwhuhan** [foldedtoad/dwm3000#16]
Hi, sir. Great work. I noticed that just three boards are surpoorted by the project beacuse three borads name are listed in the CMake.txt. Here, we'd like to know if we can Implement the code on the new DWM3001C model or not, which integrates a Nordic NRF52833 chip? 
If we can, could you please share how to do it any places in the project need to be changed to make it work on the NRF52833. thank you so much and look forward to your messages (Email: zyliu0017@whu.edu.cn).

**Comment by robogrow** [comment]
yes we have several large UWB project that need to support iphones and samsung phones. Would like to implement a FIRA compliant that is equivalent the the Nearby Interaction demo on the dwm2001cdk and also provides distant, angle and azimuth data on the J20 port of the qorvo board. 
If it can be done on the ardin bard that would work as well.  One question is the second bard required because Zephyr does run on the nrfsoc? peter@ccaway.net Have three UWB project with large DOTs going n now. Also, have rbotic mowing system using RTK GPS would like to replace with cheaper UWB. Ceva claim they can get 1 cm accuracy. Would like to see if this is possible with UWB, IMU and wheel tick lots of anchor  fusion. ie. least squares. Seen an article of increased accuracy with more anchors for sports applications. Can call me at 301 916-5722 peter@ccaway.net

**Comment by zyliuwhuhan** [comment]
Hi, 




This is Zuoya, a young researcher with UWB chip, thank you so much for your messages. What's the best ways for you to communicate, WhatsApp is OK? 




Zuoya


-----Original Messages-----
From:robogrow ***@***.***>
Sent Time:2022-11-16 08:04:23 (Wednesday)
To: foldedtoad/dwm3000 ***@***.***>
Cc: "zuoya.liu" ***@***.***>, Author ***@***.***>
Subject: Re: [foldedtoad/dwm3000] Can we Implement the code on the new DWM3001C model? (Issue #16)






yes we have several large UWB project that need to support iphones and samsung phones. Would like to implement a FIRA compliant that is equivalent the the Nearby Interaction demo on the dwm2001cdk and also provides distant, angle and azimuth data on the J20 port of the qorvo board.
If it can be done on the ardin bard that would work as well. One question is the second bard required because Zephyr does run on the nrfsoc? ***@***.*** Have three UWB project with large DOTs going n now. Also, have rbotic mowing system using RTK GPS would like to replace with cheaper UWB. Ceva claim they can get 1 cm accuracy. Would like to see if this is possible with UWB, IMU and wheel tick lots of anchor fusion. ie. least squares. Seen an article of increased accuracy with more anchors for sports applications. Can call me at 301 916-5722 ***@***.***
